### ISSUE TITLE
Some mailers may require a Revocation List

### ISSUE THREAD

**Issue by travispessetto** [travispessetto/OrigamiSMTP#15]
When trying to use Origami STMP I got the following error from a C# application:

The server's SSL certificate could not be validated for the following reasons:
• The server certificate has the following errors:
  • The revocation function was unable to check revocation for the certificate.

from MailKit 2.14.0

Code used for sending mail from Mailkit:
```csharp
using (var client = new MailKit.Net.Smtp.SmtpClient())
          {
                // note from the documentation, if 587 is a port, using the Auto socket option will choose the correct SSL mode. -cjd 
                client.Connect(networkCredential.Domain, Port, MailKit.Security.SecureSocketOptions.Auto);
                client.Authenticate(networkCredential.UserName, networkCredential.Password);
                client.Send(mail);
                client.Disconnect(true);
           }
```

**Comment by travispessetto** [comment]
Here is my guess as to why it doesn't work:

- The certificate does not have a revocation list URL
- The server does not have a in-built server for revocation list

Proposed solution:

- Create a new project (Origami Revocation) using Spring Boot
- Create a new CA certificate to integrate into Origami SMTP
-  Integrate Origami Revocation into Origami GUI under a new thread

**Comment by travispessetto** [comment]
There may be another problem where localhost is not a valid domain.  Therefore I bought the domain name origamimail.us.  The subdomain local.origamimail.us will resolve to 127.0.0.1 in hopes that this helps with issues.  This means instead of using localhost local.origamimail.us will be used.

**Comment by travispessetto** [comment]
This appears to be fixed by commit 760a0fa6de73e33345203d76f88fe7686a0b6a70.  A new root CA will need to be installed on client machine.

Commit:
- Changes servername to local.origamimail.us
- Changes certificates have a CRL located at origami.pessetto.com/revocation.crl
